### ISSUE TITLE
Create a script for a virtual Python env to run benchmark tasks

### ISSUE THREAD

**Issue by karlaspuldaro** [qiskit-community/submit-metriq#2]
Subtask of #1 

The idea is to create a script that will be run one time to gather historical data for average of circuit depth and gate count, using experiment code documented [here](https://github.com/qiskit-community/submit-metriq/issues/1#issuecomment-1507586579).
Once the backfill is complete, this structure can be used to run any benchmark tasks on newer qiskit versions. This automation on future qiskit releases will be implemented as a separate PR.

- Get all qiskit versions from pip command `pip index versions qiskit `and filter those since 2020-03-24 (last update in metriq), hard code it for one time run to build historical data.
- The entry point is a script that creates on tox environment per qiskit version. Input parameter is qiskit version (later on, also python version and the benchmark script - since in the future there can be other tasks).
- The script creates a tox environment with the given qiskit version, loads the benchmark script, run it N times (starting at 100) and outputs the average of circuit depth and gate count (first print to screen, later it can output to a file).
- Once the job is complete, the environment is shut down and cleaned.

**Comment by karlaspuldaro** [comment]
@1ucian0 
`pip index versions qiskit` lists all major and minor versions, is it ok to use them all (since March 2020)?

**Comment by 1ucian0** [comment]
Yes! consider using `pip index versions qiskit-terra` instead. No need to worried about patch versions indeed. You can also use https://pypi.org/pypi/qiskit-terra/json, in case you prefer.
